docs/why_ibis.md

```diff
@@ -0,0 +1,65 @@
+---
+hide:
+  - footer
+---
+
+# Why try Ibis?
+
+Ibis is a dataframe interface to execution engines with support for [15+
+backends](./backends/index.md). Ibis doesn't replace your existing execution
+engine, it _extends_ it with powerful abstractions and intuitive syntax.
+
+You are probably already working with one or more `$BIG_DATA_QUERY_ENGINE` and
+whether you're an individual or an enterprise, changing deployments can be a
+massive headache.
+
+For an individual, you're already doing everything yourself. Changing up your
+workflow and moving all your data around is a pain.
+
+If you work at a large enterprise, you probably don't have the option to change
+what execution resources are available to you.
```

Comment:
I don't know if we want to presume what the work environment is like for potential users. If a user's experience doesn't match this, they may not think ibis is for them. I think we can go hard on "ibis wraps your existing tools to make them more useful" front without presuming they're doing big data or working at a large inflexible corporation.

Reply:
Yep, my biases are showing and I don't know that this adds much.